### PR TITLE
store econ committee questions off-chain

### PR DESCRIPTION
**Pull request by turadg**
## Description

Publish committee questions to chain storage.

### Security Considerations

Gives contracts a Marshaller from the board. It is read-only.

### Documentation Considerations

It's a feature of `@agoric/governance` contracts but also `@agoric/inter-protocol`. Documented in both READMEs.

### Testing Considerations

Unit and swingset tests pass, but they don't cover `startEconomicCommittee`. So I did a chain test per the "Demo" section of the README. The contract publishes 'undefined' as the first question so the key is present:
```
❯ agd query vstorage keys 'published.committees'
children:
- Initial_Economic_Committee
pagination: null
❯ agd query vstorage keys 'published.committees.Initial_Economic_Committee'
children:
- latestQuestion
pagination: null
❯ agoric follow :published.committees.Initial_Economic_Committee.latestQuestion
SES Lockdown using options from environment variables "LOCKDOWN_ERROR_TAMING"
undefined
```

### CODE REVIEW

**Review by Chris-Hibbert — COMMENTED** [review]
I don't see any test output changes showing evidence that questions were ever recorded. Did I miss it somehow?  I'm sure there are tests in which questions are asked.

**Comment by dckc** [comment]
A test showing a question to the econ committee to pause one side of the PSM would be most welcome!

**Comment by Chris-Hibbert** [comment]
> A test showing a question to the econ committee to pause one side of the PSM would be most welcome!

Well, it turns out that in service of #6007, I'm working on a test showing a governed PSM setting the filter.  The existing tests work with a bare PSM that doesn't actually have governance installed AFAICT.   (I didn't know that was possible!)

**Comment by turadg** [comment]
> I don't see any test output changes showing evidence that questions were ever recorded. Did I miss it somehow?

Nope. I tried to call this state out in Testing Plan.

> I'm sure there are tests in which questions are asked.

If that's sufficient: e91c30161a2a8b7ad2396c4146092a7f27b628b5

**Review by Chris-Hibbert — APPROVED** [review]
> > I'm sure there are tests in which questions are asked.

> If that's sufficient: https://github.com/Agoric/agoric-sdk/commit/e91c30161a2a8b7ad2396c4146092a7f27b628b5

That's what I was looking for, thanks.

File: packages/inter-protocol/src/proposals/econ-behaviors.js [hunk 1]
```diff
@@ -95,16 +95,25 @@ export const startEconomicCommittee = async (
   },
   { options: { econCommitteeOptions = {} } = {} },
 ) => {
+  const STORAGE_PATH = 'committee';
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/econ-behaviors.js]: This suggests it's the only committee. That's true to some extent, but I wonder to what extent we should commit to that. What do you think about `published.committee.economicGovernance` or just `published.committee1`?

Comment (turadg) [reply packages/inter-protocol/src/proposals/econ-behaviors.js]: Good call. Updated to be `published.committees.Initial_Economic_Committee`, sanitizing the `committeeName` property.

**Comment by turadg** [comment]
@Mergifyio requeue

**Pull request merged**